Getting source: Add --depth 1 arguments (#4994)
godotengine/godot-docs: development/compiling/getting_source.rst

::

>     # Use --depth 1 argument to avoid cloning the whole unneeded commit history.
>     # Don't use --depth 1 if you're cloning for engine development purposes.
>     git clone https://github.com/godotengine/godot.git --depth 1

Reviewer: ```suggestion
    git clone https://github.com/godotengine/godot.git
    # You can add the --depth 1 argument to omit the commit history.
    # Faster, but not all Git operations (like blame) will work.
```

For any stable release, visit the `release page <https://github.com/godotengine/godot/releases>`__
and click on the link for the release you want.
You can then download and extract the source from the download link on the page.

With ``git``, you can also clone a stable release by specifying its branch or tag
after the ``--branch`` (or just ``-b``) argument::

    # Clone the continuously maintained stable branch (`3.x` as of writing).

>     git clone https://github.com/godotengine/godot.git --branch 3.x --depth 1

Reviewer: ```suggestion
    git clone https://github.com/godotengine/godot.git --branch 3.x
```

    # Clone the `3.2.3-stable` tag. This is a fixed revision that will never change.

>     git clone https://github.com/godotengine/godot.git -b 3.2.3-stable --depth 1

Reviewer: ```suggestion
    git clone https://github.com/godotengine/godot.git -b 3.2.3-stable
```

::

    git clone https://github.com/godotengine/godot.git
    # You can add the --depth 1 argument to omit the commit history.
    # Faster, but not all Git operations (like blame) will work.

For any stable release, visit the `release page <https://github.com/godotengine/godot/releases>`__
and click on the link for the release you want.
You can then download and extract the source from the download link on the page.

With ``git``, you can also clone a stable release by specifying its branch or tag
after the ``--branch`` (or just ``-b``) argument::

    # Clone the continuously maintained stable branch (`3.x` as of writing).

>     git clone https://github.com/godotengine/godot.git --branch 3.x

Reviewer: ```suggestion
    git clone https://github.com/godotengine/godot.git -b 3.x
```
